state.go

```diff
@@ -13,4 +13,6 @@ type State struct {
 	Pid int `json:"pid"`
 	// BundlePath is the path to the container's bundle directory.
 	BundlePath string `json:"bundlePath"`
+	// Annotations holds runtime-specified information that is not structured by this specification.
```

Comment:
On Mon, Jan 04, 2016 at 11:45:11AM -0800, Vish Kannan wrote:
> > +	// Annotations holds runtime-specified information that is not structured by this specification.
> 
> But what prevents users from adding their own metadata here?

There are no technical limits to something like [1], but with a
required BundlePath there is no need [2].  My personal preference
would be to drop the BundlePath requirement [3], but addressing that
seems like it's outside the scope for this PR.

[1]: https://github.com/opencontainers/specs/pull/188#issuecomment-146912525
[2]: https://github.com/opencontainers/specs/pull/188#issuecomment-147026995
[3]: https://github.com/opencontainers/specs/pull/188#issuecomment-147030855